runtime/lua/vim/loader.lua

```diff
@@ -1,5 +1,5 @@
 local uv = vim.uv
-local uri_encode = vim.uri_encode
+local uri_encode = vim.uri.encode
```

Comment:
If the loader now depends on the uri module then we should inline the uri module into the nvim binary.

This would make my other comment moot since requiring it essentially becomes free.